src/System.Diagnostics.PerformanceCounter/src/PinvokeAnalyzerExceptionList.analyzerdata.netstandard

```diff
@@ -0,0 +1,4 @@
+kernel32.dll!CreateFileMappingW
```

Comment:
We will need to decide if we want to support this library on UWP or not. If we do we need to get exceptions for these or change the APIs.

Reply:
Not right now. I've asked for these exclusion files rather than inline disabling the warnings so it's obvious there's violations.